lib/gridstore/src/gridstore.rs

```diff
@@ -543,7 +520,7 @@ impl<V: Blob> Gridstore<V> {
                 length,
             } = pointer;
 
-            let raw = self.read_from_pages(page_id, block_offset, length);
+            let raw = self.read_from_pages::<true>(page_id, block_offset, length);
```

Comment:
This is the main change (which could've been done without the refactor, but it was a quick change 😸)